compute_endpoint/globus_compute_endpoint/endpoint/endpoint.py

```diff
@@ -565,13 +567,26 @@ def start_endpoint(
                 no_color=no_color,
             )
 
+            if not ep_info:
+                ep_info = {
+                    "posix_pid": os.getpid(),
+                    "posix_uid": os.getuid(),
+                    "posix_gid": os.getgid(),
+                    "posix_username": pwd.getpwuid(os.getuid()).pw_name,
+                    "config.yaml": endpoint_config.source_content,
+                }
+            now_tz = datetime.now().astimezone()
+            ep_info["start_iso"] = now_tz.isoformat()
```

Comment:
Wasn't sure if start_ or started_ is better wording but start_iso is shorter.

Reply:
Heh.  Bikeshed to me.  Whichever the team likes.  FWIW, "start" was short for starttime in my mind when I first wrote that.